docs: improve how to include rouge stylesheets (#7752)
jekyll/jekyll: docs/_docs/liquid/tags.md

In order for the highlighting to show up, you’ll need to include a highlighting
stylesheet. For Pygments or Rouge you can use a stylesheet for Pygments, you

> can find an example gallery [here](https://github.com/richleland/pygments-css) or [here](http://jwarby.github.io/jekyll-pygments-themes/languages/ruby.html).

Reviewer: Let's give a higher priority to the Jekyll-oriented repo instead:

```suggestion
can find an example gallery [here](https://jwarby.github.io/jekyll-pygments-themes/languages/ruby.html) or from [its repository](https://github.com/jwarby/jekyll-pygments-themes).
```

Author: That’s fine, the previous example was visual but didn’t have readily available CSS so I kept it first. They are very similar otherwise

> Copy the CSS file (native.css for example)into your css directory. Import the syntax highlighter styles into your `main.css`

Reviewer: ```suggestion
Copy the CSS file (`native.css` for example) into your css directory and import the syntax highlighter styles into your `main.css`:
```

```css

> @import(native.css);

Reviewer: I think we're using this style elsewhere in the docs
```suggestion
@import "native.css";
```

Author: Tried doing this from my phone but now getting diff outdated message. I knew as soon as my head hit pillow last night that I had bad syntax here.

Reviewer: The correct syntax involves a String and can be either of the following:
```css
@import "native.css";
```
```css
@import url("native.css");
```
